pages/apis/rest_api/builds.md

```diff
@@ -18,7 +18,7 @@ API requests that affect a single build accept the more human readable build num
 
 ## List all builds
 
-Returns a [paginated list](<%= paginated_resource_docs_url %>) of all builds across all the user's organizations and pipelines.
+Returns a [cursor-paginated list](<%= cursor_paginated_resource_docs_url %>) of all builds across all the user's organizations and pipelines.
 If using token-based authentication the list of builds will be for the authorized organizations only.
```

Comment:
Since we're in the error, add missing comma:

```suggestion
If using token-based authentication, the list of builds will be for the authorized organizations only.
```